src/test/java/uk/gov/justice/digital/delius/service/ConvictionService_GetProbationStatusTest.java

```diff
@@ -165,13 +151,14 @@ public void canGetProbationStatusForPreviouslyKnownOffenderWithPreSentenceActivi
         when(offenderRepository.findByCrn(CRN)).thenReturn(Optional.of(offender));
         // PREVIOUSLY_KNOWN if currentDisposal is false and they have at least 1 event with a disposal
         when(offender.getCurrentDisposal()).thenReturn(0L);
+        when(offender.getActiveEvents()).thenReturn(List.of(event, event2));
         when(offender.getEvents()).thenReturn(List.of(event, event2));
```

Comment:
Did you forget to drop this line?

Reply:
Unfortunately not.  To work out the probation status and previously known termination date then we need to look at all the events, so therefore the code does indeed call both.